meagea-api/src/test/java/project/PromotionServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package project;
+
+
+import entity.Animal;
+import entity.Promotion;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+import project.dto.PromotionForm;
+import project.dto.SimplePromotionDto;
+import project.repository.AnimalFileRepository;
+import project.repository.AnimalRepository;
+import project.repository.PromotionRepository;
+import project.service.PromotionService;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class PromotionServiceTest {
+
+    @Mock
+    PromotionRepository proRepo;
+    @Mock
+    private AnimalRepository animalRepo;
+    @Mock
+    private AnimalFileRepository fileRepo;
+    @InjectMocks
+    private PromotionService service;
+
+    @Test
+    public void savePromotionTest() throws IOException {
+        Animal animal = new Animal("머핀", 5, "암컷", 3.5, true, "친칠라",
+                                    "동네", 2, 1, 2, 1);
+        given(animalRepo.findById(anyInt())).willReturn(Optional.of(animal));
```

Comment:
https://jdm.kr/blog/222

```suggestion
        given(animalRepo.findById(eq(1))).willReturn(Optional.of(animal));
```